styles/styles.css

```diff
@@ -188,6 +188,49 @@ main .section.wide {
   max-width: var(--wide-page-width);
 }
 
+main .section.formatted-text * {
+  letter-spacing: normal;
+  line-height: var(--line-height-m);
+  color: var(--body-color);
+}
+
+main .section.formatted-text h3 {
+  letter-spacing: 0.5px;
+  font-weight: var(--font-weight-bold);
+}
+
+main .section.width-67-percent {
```

Comment:
Can we use an author-friendly term for this? And how does this look on mobile? Not sure that it should translate down to 67% at 300px

Reply:
update name and added style for smaller screens